src/src/utils/api.ts

```diff
@@ -0,0 +1,16 @@
+import store from "../store/index";
+
+export const api = {
+    methods: {
+      fetchMediakiwiAPI(listID: string, itemID: string, queryString: string) {
+        let request = {
+          queryString: queryString,
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8211123)
```suggestion
          queryString,
```